wiring/inc/spark_wiring_i2c.h

```diff
@@ -105,9 +105,9 @@ class TwoWire : public Stream
   void end();
   uint8_t endTransmission(void);
   uint8_t endTransmission(uint8_t);
-  size_t requestFrom(uint8_t, size_t);
-  size_t requestFrom(uint8_t, size_t, uint8_t);
-  size_t requestFrom(const WireTransmission& transfer);
+  ssize_t requestFrom(uint8_t, size_t);
```

Comment:
I understand the idea, but this is a breaking change. The documentation states:
> Returns: byte : the number of bytes returned from the slave device. If a timeout occurs, will return 0.

Do we need to change the API here? It should still be possible to capture errors by checking for `== 0`.